packages/plugin-ext-vscode/src/node/plugin-vscode-init.ts

```diff
@@ -30,7 +31,7 @@ let pluginApiFactory: PluginAPIFactory;
 
 export const doInitialization: BackendInitializationFn = (apiFactory: PluginAPIFactory, plugin: Plugin) => {
     pluginsApiImpl.set(plugin.model.id, createVSCodeAPI(apiFactory, plugin));
-    plugins.push(plugin);
+    plugins.push({ ...plugin, pluginFolder: realpathSync(plugin.pluginFolder) });
```

Comment:
Suggestion: I don't believe this is necessary anymore, given that we already store the resolved plugin path in the `plugin.pluginFolder`.